transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDomainSocketChannel.java

```diff
@@ -37,6 +37,13 @@ public EpollDomainSocketChannel(Channel parent, FileDescriptor fd) {
         super(parent, fd.intValue());
     }
 
+    /**
+     * Creates a new {@link EpollDomainSocketChannel} from an existing {@link FileDescriptor}
+     */
+    public EpollDomainSocketChannel(FileDescriptor fd) {
```

Comment:
Because we have an extra "abstraction" for FileDescriptor ;) Also I will move NativeFileDescriptor to transport-native module once created so it will be not epoll specific anymore